install/migrations/update_10.0.5_to_10.0.6/unmanaged.php

```diff
@@ -0,0 +1,172 @@
+$migration->addKey('glpi_unmanageds', 'groups_id_tech');
+
+// add default rules for unmanaged device
+$rules[] = [
+    'name'      => 'Unmanaged update (by name)',
+    'uuid'      => 'glpi_rule_import_asset_unmanaged_update_name',
+    'match'     => 'AND',
+    'is_active' => 1,
+    'criteria'  => [
+        [
+            'criteria'  => 'itemtype',
+            'condition' => Rule::PATTERN_IS,
+            'pattern'   => 'Unmanaged'
+        ],
+        [
+            'criteria'  => 'name',
+            'condition' => Rule::PATTERN_EXISTS,
+            'pattern'   => 1
+        ],
+        [
+            'criteria'  => 'name',
+            'condition' => Rule::PATTERN_FIND,
+            'pattern'   => 1
+        ]
+    ],
+    'action'    => [
+        [
+            'field'         => '_inventory',
+            'action_type'  => "assign",
+            'value'         => RuleImportAsset::RULE_ACTION_LINK_OR_IMPORT,
+        ]
+    ]
+];
+
+$rules[] = [
+    'name'      => 'Unmanaged import (by name)',
+    'uuid'      => 'glpi_rule_import_asset_unmanaged_import_name',
+    'match'     => 'AND',
+    'is_active' => 1,
+    'criteria'  => [
+        [
+            'criteria'  => 'itemtype',
+            'condition' => Rule::PATTERN_IS,
+            'pattern'   => 'Unmanaged'
+        ],
+        [
+            'criteria'  => 'name',
+            'condition' => Rule::PATTERN_EXISTS,
+            'pattern'   => 1
+        ]
+    ],
+    'action'    => [
+        [
+            'field'         => '_inventory',
+            'action_type'  => "assign",
+            'value'         => RuleImportAsset::RULE_ACTION_LINK_OR_IMPORT,
+        ]
+    ]
+];
+
+$rules[] = [
+    'name'      => 'Unmanaged import denied',
+    'uuid'      => 'glpi_rule_import_asset_unmanaged_import_denied',
+    'match'     => 'AND',
+    'is_active' => 1,
+    'criteria'  => [
+        [
+            'criteria'  => 'itemtype',
+            'condition' => Rule::PATTERN_IS,
+            'pattern'   => 'Unmanaged'
+        ]
+    ],
+    'action'    => [
+        [
+            'field'         => '_inventory',
+            'action_type'  => "assign",
+            'value'         => RuleImportAsset::RULE_ACTION_DENIED
+        ]
+    ]
+];
+
+$query = "SELECT MAX(`ranking`) FROM `glpi_rules`";
+$result = $DB->query($query);
+$ranking   = $DB->result($result, 0, 0);
+
+foreach ($rules as $rule) {
+    $rulecollection = new RuleImportAssetCollection();
+    $input = [
+        'is_active' => $rule['is_active'],
+        'name'      => $rule['name'],
+        'uuid'      => $rule['uuid'],
+        'match'     => $rule['match'],
+        'sub_type'  => RuleImportAsset::getType(),
+        'ranking'   => $ranking
+    ];
+
+    if ($rulecollection->getFromDBByCrit(['uuid' => $rule['uuid']])) {
+        //rule already exists with uuid, ignore.
+        continue;
+    }
+
+    // Add rule
+    $query = "INSERT INTO `glpi_rules`
```

Comment:
Maybe better to use buildInsert rather tahn prodiuce raw SQL. BTW, isn't there any existing method in migration stuff to add rules?

Reply:
done with https://github.com/glpi-project/glpi/pull/13326/commits/978615f6313c5ff73b2a81c8208526c61c3e6104